news/2025/2025-05-03-expert-global-taiko-showdown-2025-concludes.md

```diff
@@ -0,0 +1,637 @@
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/f4r-YGyv7k4" frameborder="0" allowfullscreen></iframe>
+</div>
+
+We're now in the first round of Semifinals for the Loser's Bracket, with a legendary lineup of players to say the least, with ::{ flag=KR }:: [Peaceful](https://osu.ppy.sh/users/165027) on one side, being known as one of the best tournament players in the game, with 4 different TWC podium finishes, and three of them being 2nd place in 2020, 2021, and 2022 all in a row, and being the current [Project Prism champion](https://osu.ppy.sh/community/forums/topics/1909279). On the other side, we have ::{ flag=JP }:: [_Kan2](https://osu.ppy.sh/users/7160196), having one of the highest skillcaps for a tournament player, despite not playing in tournaments, showcased with his 2x miss on [TremENDouS](https://osu.ppy.sh/beatmapsets/1632807#taiko/3332914), EGTS 2021's Grand Finals Tiebreaker, being unmatched to this day, and being the only player that has won both Squad Global Taiko Showdowns to ever happen, in 2021, and 2023.
+
+Now, how could this match between such juggernauts of osu!taiko go? Picks for both players were pretty unsurprising, with ::{ flag=KR }:: [Peaceful](https://osu.ppy.sh/users/165027) choosing the reading aspect of the mappool, with [FM3](https://osu.ppy.sh/beatmapsets/2336583#taiko/5016186), [HR2](https://osu.ppy.sh/beatmapsets/2336561#taiko/5016142), [HD1](https://osu.ppy.sh/beatmapsets/2336375#taiko/5015685), [FM1](https://osu.ppy.sh/beatmapsets/2336556#taiko/5016129), [NM4](https://osu.ppy.sh/beatmapsets/2336545#taiko/5016102), and [NM6](https://osu.ppy.sh/beatmapsets/2336555#taiko/5016128), as a mechanics-oriented final pick. For ::{ flag=JP }:: [_Kan2](https://osu.ppy.sh/users/7160196), we ended up with [NM1](https://osu.ppy.sh/beatmapsets/2336548#taiko/5016113), [NM3](https://osu.ppy.sh/beatmapsets/2336550#taiko/5016115), [NM5](https://osu.ppy.sh/beatmapsets/2336559#taiko/5016136), [HR1](https://osu.ppy.sh/beatmapsets/2336376#taiko/5015688), [NM2](https://osu.ppy.sh/beatmapsets/2336557#taiko/5016130), and [DT2](https://osu.ppy.sh/beatmapsets/2336377#taiko/5015689) all being picked. This entire match went as according for the first 4 picks, but ::{ flag=KR }:: [Peaceful](https://osu.ppy.sh/users/165027) then lost on HD1, losing the slow Hidden reading for the second match in a row, and giving ::{ flag=JP }:: [_Kan2](https://osu.ppy.sh/users/7160196) the first break point in this match. Sadly for him though, this would not be the last break point in the match, as ::{ flag=KR }:: [Peaceful](https://osu.ppy.sh/users/165027) ended up retaliating, by first winning [HR1](https://osu.ppy.sh/beatmapsets/2336376#taiko/5015688) in dominating fashion by FC'ing it, and getting to 4-4, and finally, winning [DT2](https://osu.ppy.sh/beatmapsets/2336377#taiko/5015689) in one of the most impressive break points to happen, with a 483 points difference, to win the whole match.
```

Comment:
this paragraph has the same issue as the other one, you are diluting the main point with just spamming all of their picks rather than writing what the skill that they weighed into was